modules/spark/src/main/scala/org/apache/ignite/spark/impl/QueryHelper.scala

```diff
@@ -105,6 +111,7 @@ private[apache] object QueryHelper {
       */
     def saveTable(data: DataFrame,
         tblName: String,
+        schemaName: Option[String],
```

Comment:
The schema name isn't necessary in the Ignite `INSERT` statement. The schema name gets passed through to `savePartition()` and then is applied to the data streamer separately from the SQL statement. So yes it will work if we have two tables with the same name in different schemas.